index.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Portfolio</title>
+</head>
+<body>
+    <nav class="toolbar">
+        <h2>Welcome</h2>
+        <button class="icon">☰</button>
+    </nav>
+    <main>
+        <section class="header">
+            <h2 class="name">Hey there. I'm Juan Sebastian</h2>
```

Comment:
Please consider changing this heading into an `h1` tag because it defines the most important heading and will also help your SEO(Search engine optimizations)